<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFFCSMSDEvaluationGetNFromFits</class>
 <widget class="QDialog" name="QFFCSMSDEvaluationGetNFromFits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCS MSD: Get Parameters from FCS fit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>evaluation to read from:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>apply to:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbApplyTo">
       <item>
        <property name="text">
         <string>current</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>all runs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>all files, this run</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>everything</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFResultsGroupComboBox" name="cmbEval"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkGetN">
       <property name="text">
        <string>get particle number N</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chkGetTriplet">
       <property name="text">
        <string>get triplet parameters</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="chkGetFocus">
       <property name="text">
        <string>get focus parameters</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFResultsGroupComboBox</class>
   <extends>QComboBox</extends>
   <header>qfresultsgroupcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFFCSMSDEvaluationGetNFromFits</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFFCSMSDEvaluationGetNFromFits</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
